src/ess/amor/resolution.py

```diff
@@ -144,9 +145,21 @@ def sigma_Q(
     :
         Combined resolution function.
     """
-    return sc.sqrt(
-        angular_resolution**2 + wavelength_resolution**2 + sample_size_resolution**2
-    ).flatten(to="detector_number").max("detector_number") * sc.midpoints(q_bins)
+    h = da.bins.concat().hist(Q=qbins)
+    s = (
+        (
+            da
+            * (
+                angular_resolution**2
+                + wavelength_resolution**2
+                + sample_size_resolution**2
+            )
+            * da.bins.coords['Q'] ** 2
+        )
+        .bins.concat()
+        .hist(Q=qbins)
+    )
+    return sc.sqrt(sc.values(s / h))
```

Comment:
I have to say I didn't follow all the math in the PR description, but I looked again at the Stahn & Glavic paper, and the old [ess docs](https://scipp.github.io/ess/instruments/amor/amor_reduction.html#Resolution-function).

Equation (5) in the paper appears to say
```
sigma_q = (P * sqrt(angular_resolution**2 + wavelength_resolution**2 + sample_size_resolution**2) * Q) / P
```
while you seem to have
```
sigma_q = sqrt(P * (angular_resolution**2 + wavelength_resolution**2 + sample_size_resolution**2) * Q**2) / P)
```
which is slightly different?

Or maybe I got lost in the binning and histogramming?

If you think your math is correct, you should check with instrument scientists.

Reply:
I think what's written in the issue description is correct, and after talking to Jochen and Nicolo I didn't get the the impression that they think otherwise.